module_11/data/authtorization.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$authtirized = false;
+
+if (array_key_exists('login', $_POST)) {
+    for ($i = 0; $i < count($users); $i++) {
+        if ($_POST['login'] === $users[$i]
+            && $_POST['password']
+            === $passwords[$i]
+        ) {
+            include "include/success_message.php";
+            $authtirized = true;
+            break;
```

Comment:
The "break" command does not work within an "if" statement.

If you remove the "break" command from your code and then test the code, you should find that the code works exactly the same without a "break" command as with one.

"Break" is designed for use inside loops (for, while, do-while, enhanced for and switch).

Reply:
I use break to stop the for loop that checks if the password is correct. How can i remake this code to avoid break?

Reply:
I think i understand!